src/xrc-tests/src/tests/get_icp_xdr_rate.rs

```diff
@@ -0,0 +1,156 @@
+use ic_xrc_types::{Asset, AssetClass, GetExchangeRateRequest, GetExchangeRateResult};
+
+use crate::{
+    container::{run_scenario, Container},
+    mock_responses,
+};
+
+#[ignore]
+#[test]
+/// Setup:
+/// * Deploy mock FOREX data providers and exchanges
+/// * Start replicas and deploy the XRC, configured to use the mock data sources
+///
+/// Runbook: Assert query for ICP/XDR exchange rate returns the expected value
+///
+/// Success criteria: All queries return the expected values
+fn get_icp_xdr_rate() {
+    let now = time::OffsetDateTime::now_utc().unix_timestamp() as u64;
+    let set1_timestamp = now / 60 * 60;
+    let set2_timestamp = set1_timestamp - 60;
+    let set3_timestamp = set2_timestamp - 60;
```

Comment:
Why are the variables called `set[number]_timestamp`?

Reply:
What's the time unit?

Reply:
Renamed the variable to `request_[number]_timestamp_seconds` to make it more clear and added some small comments to point the request sets.